.travis.yml

```diff
@@ -35,11 +35,9 @@ script:
     - pylint queue-health/weekly_commit_stats.py  # TODO(fejta): all python files
     - ./jenkins/diff-job-config-patch.sh
     - ./jenkins/diff-e2e-runner.sh
-    - make -C gcsweb test
```

Comment:
It already builds and tests with bazel. The only thing it was doing previously was gofmt and govet, but now that's unnecessary.